### DISCUSSION TITLE
Support multiple milestones per issue

### DISCUSSION THREAD

**Discussion by lukaseder** [community/community#4566] [Projects and Issues]
When I backport a bugfix, I can't currently assign multiple milestones to an issue. Many other issue trackers have this feature, e.g. JIRA. I currently create additional issues and manually reference between the issues to make sure the release notes track this correctly. But it seems like a low hanging fruit from a usability perspective to be able to assign an issue to multiple milestones.

The devil is in the details. A few attributes would probably be milestone specific, e.g. whether an issue is open/closed. There are various ways to solve this. Core idea is, I want to be able to formally track my backports.

**Comment by MSNev** [comment]
Not an answer, this is just a +100 as there are lots of other activity on this in the "deprecated" issue repo as well 
https://github.com/isaacs/github/issues/797

**Comment by tisonkun** [comment]
+1 for this feature and here is another earlier issue https://github.com/isaacs/github/issues/441 .

We do patch releases in multiple minor release lifetime, and do really want this feature so that we don't have to use issue labels "affects-", "fixes-" to simulate it.

**Comment by ktaeyln** [comment]
Definitely a required feature to take this platform seriously for project management....

**Comment by abelsromero** [comment]
I am surprised this is not getting more attention. Multiple branch management is not easy with default GH features. :cry:

**Reply by abelsromero** [reply]
To add more about the user experience. I am working in 2 ways:
- Create tags for each branch: this means a lot of tags since we need one for each minor for each branch. The tag selector from UI become basically un-usable for normal tasks because common tagas (docs, bug, question, etc.) become buried.
- Use third-party tools: this is only available in corporate environments, not for OS.

**Comment by maxsxu** [comment]
This is definitely a meaningful feature for multi branch management. Do we have any plan to support this feature?

**Comment by SwuduSusuwu** [comment]
+22 from https://github.com/orgs/community/discussions/6804 .

Have numerous Milestones (most of the Milestones -- https://github.com/SwuduSusuwu/SubStack/milestones -- refer to complex features which reference numerous issues,) too complex to summarize as 1 or 2 word "tags".
Thought that Github allowed to stick issues to multiple milestones,
have put most issues to 2 or more milestones,
but now found that undid the issues bond to previous milestones.
Most of the issues -- https://github.com/SwuduSusuwu/SubStack/issues -- exist as "tasklets"/submodules of 2 or more milestones.

Cool if multiple milestones can reuse issues (thus once those issues close, if their milestones have no other issues, Github lists that such milestone closes too.)

**Comment by VSXMSX** [comment]
got same requirement, now starting to clog our GitHub with even more labels
